### PR TITLE
[Improvement][Doc] Add doc about varPool support syntax #{setValue(key=value)}

### PR DESCRIPTION
**Pull request by exmy**
## Purpose of the pull request

Add the doc about https://github.com/apache/dolphinscheduler/issues/9576

### CODE REVIEW

File: docs/docs/en/guide/parameter/context.md [hunk 1]
```diff
@@ -55,7 +55,15 @@ There is only the "id" value. Although the user-defined SQL query both "id" and
 ### SHELL
 
 `prop` is user-specified and the direction is `OUT`. The output is defined as an export parameter only when the direction is `OUT`. Choose data structures for data type according to the scenario, and leave the value part blank.
-The user needs to pass the parameter when creating the shell script, the output statement format is `${setValue(key=value)}`, the key is the `prop` of the corresponding parameter, and value is the value of the parameter.
+
+The user needs to pass the parameter when creating the shell script, the output statement format is `${setValue(key=value)}` or `#{setValue(key=value)}`, the key is the `prop` of the corresponding parameter, and value is the value of the parameter.
```
Comment (SbloodyS) [inline docs/docs/en/guide/parameter/context.md]:
```suggestion
If the user needs to pass the parameter when creating the shell script. The output statement format is `${setValue(key=value)}` or `#{setValue(key=value)}`. the key is the `prop` of the corresponding parameter. And value is the value of the parameter.
```

File: docs/docs/zh/guide/parameter/context.md [hunk 2]
```diff
@@ -57,13 +57,19 @@ prop 为用户指定；方向选择为 OUT，只有当方向为 OUT 时才会被
 prop 为用户指定；方向选择为 OUT，只有当方向为 OUT 时才会被定义为变量输出；数据类型可以根据需要选择不同数据结构；value 部分不需要填写。
 
 
-用户需要传递参数，在定义 shell 脚本时，需要输出格式为 ${setValue(key=value)} 的语句，key 为对应参数的 prop，value 为该参数的值。
+用户需要传递参数，在定义 shell 脚本时，需要输出格式为 `${setValue(key=value)}` 或者 `#{setValue(key=value)}` 的语句，key 为对应参数的 prop，value 为该参数的值。
```
Comment (SbloodyS) [inline docs/docs/zh/guide/parameter/context.md]:
```suggestion
如果用户需要传递参数，在定义 shell 脚本时，需要输出格式为 `${setValue(key=value)}` 或者 `#{setValue(key=value)}` 的语句，key 为对应参数的 prop，value 为该参数的值。
```

**Review by SbloodyS — CHANGES_REQUESTED** [review]
Do you think it would be better?

File: docs/docs/en/guide/parameter/context.md (hunk 1 above)
Comment (exmy) [reply docs/docs/en/guide/parameter/context.md]: Maybe we can change it to this: 

"If the user wants to pass parameters, you need to output statements in the form of `${setValue(key=value)}` or `#{setValue(key=value)}` when defining the shell script, where the key is the `prop` of the corresponding parameter, and value is the value of the parameter.

如果用户想要传递参数，那么在定义 shell 脚本时，需要输出格式为 `${setValue(key=value)}` 或者 `#{setValue(key=value)}` 的语句，其中 key 为对应参数的 prop，value 为该参数的值。"

What do you think?

Comment (SbloodyS) [reply docs/docs/en/guide/parameter/context.md]: The chinese part LGTM. But I think the English part of ```suggestion change``` is better. @exmy

**Comment by zhongjiajie** [comment]
Hi, we are going to release dolphinscheduler version 300-beta-1, and this should be included. It is seem this PR not update for some time, so I will make some change to it, hope you do not mind @exmy

**Comment by zhongjiajie** [comment]
Do not merge until #10119 is be merged

**Comment by zhongjiajie** [comment]
close this because already added in #10245

**Pull request closed without merging**